core/server/src/main/java/alluxio/worker/file/FileDataManager.java

```diff
@@ -176,32 +176,34 @@ public void lockBlocks(long fileId, List<Long> blockIds) throws IOException {
       if (mPersistingInProgressFiles.containsKey(fileId)) {
         throw new IOException("the file " + fileId + " is already being persisted");
       }
-      try {
-        // lock all the blocks to prevent any eviction
-        for (long blockId : blockIds) {
-          long lockId = mBlockWorker.lockBlock(Sessions.CHECKPOINT_SESSION_ID, blockId);
-          blockIdToLockId.put(blockId, lockId);
-        }
-      } catch (BlockDoesNotExistException e) {
-        errors.add(e);
-        // make sure all the locks are released
-        for (long lockId : blockIdToLockId.values()) {
-          try {
-            mBlockWorker.unlockBlock(lockId);
-          } catch (BlockDoesNotExistException bdnee) {
-            errors.add(bdnee);
-          }
+    }
+    try {
+      // lock all the blocks to prevent any eviction
+      for (long blockId : blockIds) {
```

Comment:
As far as I know, I don't think we use write lock for block. Lock is only necessary after commitBlock (inclusive).  I can rename it or you can do it if you have a name suggestion accepted by the team. I am fine with renaming.